README.md

```diff
@@ -27,41 +58,19 @@ Example:
 git clone https://github.com/awslabs/fhir-works-on-aws-deployment.git
 ```
 
-**Note**: To modify FHIR Works on AWS, create your own fork of the GitHub repository. This allows you to check in any changes you make to your private copy of the code.
+**Note**: To modify FHIR Works on AWS, create your own fork of the GitHub repository. This allows you to check in any changes you make to your private copy of the code.  See [CONTRIBUTING](./CONTRIBUTING.md) for more details.
 
-2. Use one of the following links to download FHIR Works on AWS:
+2. Use one of the following links to install FHIR Works on AWS:
 
 - [Linux/macOS](./INSTALL.md#linux-or-macos-installation)
 - [Windows](./INSTALL.md#windows-installation)
 - [Docker](./INSTALL.md#docker-installation)
 
-
 3. Refer to these [instructions](./DEVELOPMENT.md) for making code changes.
 
-If you intend to use FHIR Implementation Guides read the [Using Implementation Guides](./USING_IMPLEMENTATION_GUIDES.md) documentation first. 
-
-If you intend to do a multi-tenant deployment read the [Using Multi-Tenancy](./USING_MULTI_TENANCY.md) documentation first. 
-
-## Architecture
-
-The system architecture consists of multiple layers of AWS serverless services. The endpoint is hosted using API Gateway. The database and storage layer consists of Amazon DynamoDB and S3, with Elasticsearch as the search index for the data written to DynamoDB. The endpoint is secured by API keys and Cognito for user-level authentication and user-group authorization. The diagram below shows the FHIR server’s system architecture components and how they are related.
- 
-![Architecture](resources/architecture.png)
-
-## Components overview
+If you intend to use FHIR Implementation Guides, read the [Using Implementation Guides](./USING_IMPLEMENTATION_GUIDES.md) documentation first. 
```

Comment:
Suggested: "For information about using FHIR Implementation Guides, see Using Implementation Guides."

Reference: https://alpha-docs-aws.amazon.com/awsstyleguide/latest/styleguide/crossreferences.html